### PR TITLE
Create new rails app with rspec

### PR DESCRIPTION
**Pull request by ahmedalbaiti**
Created the basic Smartflix app with a controller and view on the root index of the app

### CODE REVIEW

**Comment by smart-jones** [comment]
You have a `spec/` directory and a `test/` directory. If your interest is in RSpec then the `test/` directory needs to go.

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "3.0.3"
```
Comment (smart-jones) [inline Gemfile]: Latest version of Ruby is *3.1.2*

File: spec/static_spec.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'index route', type: :system do
+    it 'shows page content' do
```
Comment (smart-jones) [inline spec/static_spec.rb]: The indentation here is off. Run `rubocop` to remove this minor offences

File: test/test_helper.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+ENV["RAILS_ENV"] ||= "test"
```
Comment (smart-jones) [inline test/test_helper.rb]: You can get rid of the `test/` directory, if you are only interested in RSpec tests

Comment (ahmedalbaiti) [reply test/test_helper.rb]: Is it not a directory that I might need down the line, even if I'm only interested in RSpec tests currently?

Comment (smart-jones) [reply test/test_helper.rb]: We don't really use MiniTest at Smart so you may be better off spending your efforts on learning RSpec; you won't regret it

**Review by smart-jones — APPROVED** [review]
All good, once the indentation has been fixed.

File: spec/static_spec.rb (hunk 2 above)
Comment (ahmedalbaiti) [reply spec/static_spec.rb]: done

File: Gemfile (hunk 1 above)
Comment (ahmedalbaiti) [reply Gemfile]: updated

**Pull request merged**